common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -244,37 +244,37 @@ def test_user_can_enroll(self):
         assert course_entitlement.enrollment_course_run is not None
 
     def test_user_can_unenroll(self):
-            course_entitlement = CourseEntitlementFactory(
```

Comment:
I didn't actually change this test at all, just fixed a quality issue with the indentation